woocommerce/woocommerce-gutenberg-products-block
# Update patterns design

## What

This is a follow-up from https://github.com/woocommerce/woocommerce-blocks/pull/11997#issuecomment-1835792760

Includes:
- Update the "Just Arrived Full Hero" pattern to match the design.
- Remove the title from the "Product Collection 3 Columns" pattern.
- Remove the title from the "Product Gallery" pattern.
- Update copy on the "Hero Product Chessboard".
- Removes the old unused `wc_blocks_patterns_content` constant.

## Why

To match the new designs.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Insert the "Just Arrived Full Hero" and check it matches the new design below FlPl7fHaGv9mpu8Nfl0joH-fi-158_32406 (the button colors may differ between themes because they are no hard-coded).
<img width="750" alt="Screenshot 2023-12-04 at 14 53 27" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/f919bb4a-850b-483d-be35-8ea12e161de9">

3. Insert the "Product Collection 3 Columns" and "Product Gallery" and check they don't have titles anymore.
4. Insert the "Hero Product Chessboard" and check the bottom-left square has the following copy:
```
Quality Materials
We use only the highest-quality materials in our products, ensuring that they look great and last for years to come.

Unique Design
From bold prints and colors to intricate details and textures, our products are a perfect combination of style and function.
```

## Screenshots or screencast



## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog
